components/base/BaseDropdown.vue

```diff
@@ -124,7 +144,7 @@ const shape = computed(
               : 'end-0 origin-top-right',
             shape === 'rounded' && 'rounded-md',
             shape === 'curved' && 'rounded-xl',
-            shape === 'full' && 'rounded-full',
+            shape === 'full' && 'rounded-xl',
```

Comment:
so now `curved` and `full` has same value, is this intended ?

Reply:
It is intended, otherwise the dropdown menu would look broken.